shared/signup/phone-number/phone-input.tsx

```diff
@@ -195,69 +340,130 @@ class _PhoneInput extends React.Component<Kb.PropsWithOverlay<Props>, State> {
     this.props.toggleShowingMenu()
   }
 
-  render() {
+  _onPrefixEnter = () => {
+    this._phoneInputRef.current.focus()
+  }
+
+  _renderCountrySelector = () => {
+    if (Styles.isMobile) {
+      return (
+        <Kb.Text
+          type="BodySemibold"
+          style={{
+            flexGrow: 1,
+            marginRight: Styles.globalMargins.xtiny,
+          }}
+        >
+          {this.state.country === ''
+            ? this.state.prefix === ''
+              ? '- Pick a country -'
+              : '- Invalid country prefix -'
+            : countryData[this.state.country].emoji + ' ' + countryData[this.state.country].name}
+        </Kb.Text>
+      )
+    }
+
     return (
       <>
+        <Kb.Text type="Body" style={{marginRight: Styles.globalMargins.xtiny}}>
```

Comment:
prefer using a `Box2` with `gap` / `gapEnd` instead of setting these margins.

Reply:
this is all duplicate, i'll reuse a single style